examples/react-native-expo/package.json

```diff
@@ -24,8 +24,8 @@
     "expo-file-system": "~13.0.3",
     "expo-status-bar": "~1.1.0",
     "preact-render-to-string": "^5.1.0",
-    "react": "17.0.1",
-    "react-dom": "17.0.1",
+    "react": "^18.1.0",
+    "react-dom": "^18.1.0",
```

Comment:
Did you test this change?

Reply:
The e2e tests used the React 17 version from this package from some weird reason. Have to still test this works

Reply:
```console
$ node -p 'require("react/package.json").version'
17.0.1
$ (cd e2e && node -p 'require("react/package.json").version')
18.2.0
```

The issue seems to be that Parcel resolves it from the cwd instead of doing it relatively to the module importing it, for some reason.